snuba/datasets/message_filters.py

```diff
@@ -69,7 +89,7 @@ def should_drop(self, message: Message[KafkaPayload]) -> bool:
 
 
 @dataclass
-class KafkaHeaderFilterWithBypass(KafkaHeaderFilter):
+class KafkaHeaderWithBypassFilter(KafkaHeaderFilter):
```

Comment:
Why this rename?

Reply:
All filter class names seem are suffixed with `Filter`. Renaming this class helps with consistency. This rename also does not change the understanding of what the pre-filter does.

Reply:
Sounds good.